app/src/main/java/com/raghav/spacedawn/network/SpaceFlightAPI.kt

```diff
@@ -11,13 +10,13 @@ interface SpaceFlightAPI {
     suspend fun getArticles(
         @Query("_start")
         articlesToSkip: Int = 0
-    ): Response<ArticlesResponse>
+    ): List<ArticlesResponseItem>
 
     @GET("articles")
     suspend fun searchArticles(
         @Query("_title_contains")
         searchQuery: String,
         @Query("_start")
         articlesToSkip: Int = 0
```

Comment:
i see these 2 methods call to the same api is it possible to merge them by using default arguments

Reply:
Will do in future updates